doc/cpp/probe_sample.rst

```diff
@@ -56,7 +56,7 @@ Mechanism state queries however will throw a ``cable_cell_error`` exception
 at simulation initialization if the requested state variable does not exist
 on the mechanism.
 
-Cable cell probe addresses that are described by a ``locset`` may generate more
+Cable cell probeset addresses that are described by a ``locset`` may generate more
```

Comment:
Technically it might be worse: `Locset`s are what's usually called a multiset, ie a set of unique keys
(called the _support_ of the multiset) and an associated multiplicity for each key. Probes should be
defined on the support, thus each location is probed only once. So, the number of points in the `locset`
might be larger than the number of unique points.